src/components/ShareMenu/ShareMenu.js

```diff
@@ -44,38 +100,62 @@ const defaultProps = {
       className: 'ta-mail-icon',
     },
   ],
-  extraSocialShareConfig: [],
+  extraSocialShareConfig: [
```

Comment:
I just thought that this default value will break IABPREF display. Can you leave the property empty  then add it in the  ShareMenu example instead.